terraform/stacks/tooling/outputs.tf

```diff
@@ -409,40 +375,6 @@ output "stemcell_secret_access_key_curr" {
   value = "${module.stemcell_user.secret_access_key_curr}"
 }
 
-/* limit check user */
-output "limit_check_username" {
```

Comment:
Looks like limit check user is still in use? Don't you still need these outputs?